files/build_templates/kube_cni.10-flannel.conflist

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
This file needs to be placed at the path /etc/cni/net.d/10-flannel.conflist. Where is the path set?

Reply:
We will look into this in our next revision